legacy_models/ResNet50_nFC.py

```diff
@@ -24,14 +24,12 @@ def weights_init_classifier(m):
         init.constant_(m.bias.data, 0.0)
```

Comment:
We might need to bring back the attribute models. Since I never actually tested it with tracking, I feel like there could be some potential upsides. (I would add the functionality since I'm the only one who worked on it).